File: android/app/src/main/java/com/sumitrack/android/domain/models/Installment.kt
package com.sumitrack.android.domain.models

import java.math.BigDecimal
import java.time.Instant

data class Installment(
    val id: String,
    val fkTenant: String,
    val fkSale: String,
    val amount: BigDecimal,
    val dueDate: Instant,
    val status: InstallmentStatus,
    val createdAt: Instant,
    val updatedAt: Instant,
    val syncStatus: SyncStatus,
)

File: backend/src/Sumitrack.Api/Models/Entities/Installment.cs
namespace Sumitrack.Api.Models.Entities;

public class Installment
{
    public Guid Id { get; set; }
    public Guid FkTenant { get; set; }
    public Guid FkSale { get; set; }
    public decimal Amount { get; set; }
    public DateTime DueDate { get; set; }
    public string Status { get; set; } = "pending";
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string SyncStatus { get; set; } = "synced";
}
